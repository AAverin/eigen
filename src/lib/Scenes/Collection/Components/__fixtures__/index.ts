export const CollectionFixture = {
  collection: {
    title: "Street Art Now",
    description:
      '<p>From <a href="https://www.someartsyimage.net/artist/jean-michel-basquiat">Jean-Michel Basquiat</a> to <a href="https://www.someartsyimage.net/artist/shepard-fairey">Shepard Fairey</a>, street artists are known for their creative expressions of rebellion.</p>',
    headerImage: "http://imageuploadedbymarketingteam.jpg",
    image: { edges: [{ node: { imageUrl: "https://defaultmostmarketableartworkincollectionimage.jpg" } }] },
  },
}
