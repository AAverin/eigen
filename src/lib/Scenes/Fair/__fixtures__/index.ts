export const FairFixture = {
  id: "sofa-chicago-2018",
  name: "SOFA CHICAGO 2018",
  image: {
    image_url: "https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/:version.jpg",
    aspect_ratio: 1,
    url: "https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/square.jpg",
  },
  organizer: {
    profile: {
      icon: {
        id: "58ff95298b3b8104783c119c",
        href: null,
        height: 279,
        width: 279,
        url: "https://d32dm0rphc51dk.cloudfront.net/B1x-CqoWO1KitQnRj7PrrA/untouched-png.png",
      },
      name: "SOFA CHICAGO",
      __id: "UHJvZmlsZTpzb2ZhLWNoaWNhZ28=",
    },
  },
  start_at: "2018-11-01T17:00:00+00:00",
  end_at: "2018-11-04T18:00:00+00:00",
  location: {
    address: "",
    address_2: "",
    coordinates: {
      lat: 41.891551,
      lng: -87.6073751,
    },
    display: "41.891972, -87.605161",
    city: "Chicago",
    state: "Illinois",
    country: "United States",
    postal_code: "60611",
  },
  __id: "RmFpcjpzb2ZhLWNoaWNhZ28tMjAxOA==",
}
