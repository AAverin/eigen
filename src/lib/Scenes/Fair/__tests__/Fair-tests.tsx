import { renderRelayTree } from "lib/tests/renderRelayTree"
import { graphql } from "react-relay"
import { fairFixture } from "../__fixtures__"
import { Fair, FairContainer as FairScreen } from "../Fair"
import { FairMoreInfoRenderer as FairMoreInfoScreen } from "../Screens/FairMoreInfo"

jest.unmock("react-relay")

// FIXME: Fix fixture data
describe("Fair", () => {
  const renderTree = () =>
    renderRelayTree({
      Component: FairScreen,
      query: graphql`
        query FairTestsQuery {
          fair(id: "sofa-chicago-2018") {
            ...Fair_fair
          }
        }
      `,
      mockData: {
        fair: fairFixture,
      },
    })

  xit("renders properly", async () => {
    expect((await renderTree()).html()).toMatchSnapshot()
  })

  xit("handles navigation to FairMoreInfo", async () => {
    const tree = await renderTree()

    const mockedPush = jest.fn()
    tree.find(Fair).instance().navigator = { push: mockedPush }

    const button = tree.find("Fair").find("[text='View more information']")
    button.props().onPress()

    expect(mockedPush).toHaveBeenCalledWith(
      expect.objectContaining({
        component: FairMoreInfoScreen,
      })
    )
  })
})
