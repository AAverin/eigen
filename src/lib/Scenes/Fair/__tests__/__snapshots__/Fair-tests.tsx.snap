// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fair renders properly 1`] = `
"<view><rctnavigator requestedtopofstack=\\"0\\"><rctnavitem initialroute=\\"[object Object]\\" title=\\"\\"><rctscrollview data=\\"[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]\\" listheadercomponent=\\"[object Object]\\" initialnumtorender=\\"10\\" maxtorenderperbatch=\\"10\\" scrolleventthrottle=\\"50\\" updatecellsbatchingperiod=\\"50\\" windowsize=\\"21\\" numcolumns=\\"1\\" viewabilityconfigcallbackpairs=\\"\\" stickyheaderindices=\\"\\"><view><view><view height=\\"620\\"><aropaqueimageview imageurl=\\"https://d32dm0rphc51dk.cloudfront.net/o0Zrtm-CgwbV1syD_33DuA/square.jpg\\" aspectratio=\\"1\\" width=\\"750\\"><view></view><view flexdirection=\\"row\\" justifycontent=\\"center\\" alignitems=\\"center\\" px=\\"2\\"><view flexdirection=\\"column\\" flexgrow=\\"1\\"><image source=\\"[object Object]\\"></image><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" textalign=\\"center\\" color=\\"white100\\" ellipsizemode=\\"tail\\">SOFA CHICAGO 2018</text><text font-family=\\"Unica77LL-Regular\\" font-size=\\"14px\\" lineheight=\\"24px\\" class=\\"sc-bdVaJa bDWFJH\\" textalign=\\"center\\" color=\\"white100\\" ellipsizemode=\\"tail\\">Nov 1st - Nov 4th</text></view></view><view></view></aropaqueimageview><view mt=\\"2\\"></view><view mx=\\"2\\"></view><view px=\\"2\\"><view m=\\"2\\" mt=\\"1\\"></view><view style=\\"justify-content: center; align-items: center; border-width: 1px; border-radius: 2px; height: 46px; background-color: rgb(51, 51, 51); opacity: 1;\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"24px\\" class=\\"sc-bdVaJa bDWFJH\\" color=\\"white\\" ellipsizemode=\\"tail\\">Save</text></view><view m=\\"1\\"></view></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view><view my=\\"2\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"24px\\" class=\\"sc-bdVaJa bDWFJH\\" color=\\"black100\\" textalign=\\"center\\" ellipsizemode=\\"tail\\">Hello World</text></view></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view flexdirection=\\"row\\" align-items=\\"base-line\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">View more information</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view><view mt=\\"2\\" width=\\"100%\\"></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view><view alignitems=\\"center\\" flexdirection=\\"row\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">Opening hours</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"6.123233995736766e-17,1,-1,6.123233995736766e-17,50,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view><view style=\\"height: 0px; overflow: hidden;\\"><view mt=\\"2\\"><view><text font-family=\\"AGaramondPro-Regular\\" font-size=\\"16px\\" lineheight=\\"20px\\" class=\\"sc-bwzfXH uTxCW\\" color=\\"black60\\" ellipsizemode=\\"tail\\">Hello World</text>


</view></view></view></view><view mt=\\"2\\" width=\\"100%\\"></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view mt=\\"1\\"><text font-family=\\"AGaramondPro-Regular\\" font-size=\\"26px\\" lineheight=\\"32px\\" class=\\"sc-bwzfXH uTxCW\\" ellipsizemode=\\"tail\\">Browse the fair</text></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view justifycontent=\\"space-between\\" flexdirection=\\"row\\" flexwrap=\\"nowrap\\"><view flexdirection=\\"row\\" align-items=\\"base-line\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">All artists</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view><view flexdirection=\\"row\\" align-items=\\"base-line\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">All exhibitors</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view><view flexdirection=\\"row\\" align-items=\\"base-line\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">All works</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view mt=\\"1\\"><view justifycontent=\\"center\\" height=\\"48\\" alignitems=\\"center\\" flexdirection=\\"row\\" p=\\"1\\" mb=\\"1\\"><image source=\\"[object Object]\\"></image><view width=\\"12\\"></view><text font-family=\\"Unica77LL-Medium\\" font-size=\\"12px\\" lineheight=\\"16px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">Find Exhibitors and Artists</text></view></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view my=\\"1\\"><view justifycontent=\\"space-between\\" alignitems=\\"center\\" flexdirection=\\"row\\" mb=\\"1\\"><view style=\\"opacity: 1;\\"><view flexdirection=\\"row\\" alignitems=\\"center\\"><view mr=\\"1\\"><aropaqueimageview imageurl=\\"Hello World\\" aspectratio=\\"1\\"></aropaqueimageview></view><view><text font-family=\\"AGaramondPro-Regular\\" font-size=\\"14px\\" lineheight=\\"18px\\" class=\\"sc-bwzfXH uTxCW\\" ellipsizemode=\\"tail\\">Hello World</text><text font-family=\\"Unica77LL-Regular\\" font-size=\\"12px\\" lineheight=\\"16px\\" class=\\"sc-bdVaJa bDWFJH\\" color=\\"black60\\" ellipsizemode=\\"tail\\">Hello World</text></view></view></view><view width=\\"102\\" height=\\"34\\" mr=\\"1\\"><view style=\\"justify-content: center; align-items: center; border-radius: 2px; background-color: rgb(0, 0, 0); border-width: 1px;\\"><view><text ellipsizemode=\\"tail\\">Follow</text></view></view></view></view><view mt=\\"1\\"><view><view style=\\"flex-direction: row;\\" accessibilitylabel=\\"Artworks Content View\\"></view></view></view><view mt=\\"2\\"><view flexdirection=\\"row\\" align-items=\\"base-line\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">View all works</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view></view></view></view></view><view><view px=\\"2\\" py=\\"1\\"><view my=\\"1\\"><view justifycontent=\\"space-between\\" alignitems=\\"center\\" flexdirection=\\"row\\" mb=\\"1\\"><view style=\\"opacity: 1;\\"><view flexdirection=\\"row\\" alignitems=\\"center\\"><view mr=\\"1\\"><aropaqueimageview imageurl=\\"Hello World\\" aspectratio=\\"1\\"></aropaqueimageview></view><view><text font-family=\\"AGaramondPro-Regular\\" font-size=\\"14px\\" lineheight=\\"18px\\" class=\\"sc-bwzfXH uTxCW\\" ellipsizemode=\\"tail\\">Hello World</text><text font-family=\\"Unica77LL-Regular\\" font-size=\\"12px\\" lineheight=\\"16px\\" class=\\"sc-bdVaJa bDWFJH\\" color=\\"black60\\" ellipsizemode=\\"tail\\">Hello World</text></view></view></view><view width=\\"102\\" height=\\"34\\" mr=\\"1\\"><view style=\\"justify-content: center; align-items: center; border-radius: 2px; background-color: rgb(0, 0, 0); border-width: 1px;\\"><view><text ellipsizemode=\\"tail\\">Follow</text></view></view></view></view><view mt=\\"1\\"><view><view style=\\"flex-direction: row;\\" accessibilitylabel=\\"Artworks Content View\\"></view></view></view><view mt=\\"2\\"><view flexdirection=\\"row\\" align-items=\\"base-line\\"><text font-family=\\"Unica77LL-Medium\\" font-size=\\"14px\\" lineheight=\\"20px\\" class=\\"sc-bdVaJa bDWFJH\\" ellipsizemode=\\"tail\\">View all works</text><view ml=\\"1\\"><rnsvgsvgview width=\\"16\\" height=\\"16\\" bbwidth=\\"16\\" bbheight=\\"16\\" minx=\\"0\\" miny=\\"0\\" vbwidth=\\"54\\" vbheight=\\"54\\" align=\\"xMidYMid\\" meetorslice=\\"0\\"><rnsvggroup proplist=\\"fill,fillOpacity,fillRule,stroke,strokeWidth,strokeOpacity,strokeDasharray,strokeDashoffset,strokeLinecap,strokeLinejoin,strokeMiterlimit\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvggroup proplist=\\"\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0,0,0,1\\" fill-opacity=\\"1\\" fill-rule=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" font=\\"[object Object]\\"><rnsvgpath proplist=\\"fill,fillRule\\" opacity=\\"1\\" matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" scalex=\\"1\\" scaley=\\"1\\" originx=\\"0\\" originy=\\"0\\" skewx=\\"0\\" skewy=\\"0\\" x=\\"0\\" y=\\"0\\" fill=\\"0,0.7607843137254902,0.7607843137254902,0.7607843137254902,1\\" fill-opacity=\\"1\\" fill-rule=\\"0\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" d=\\"M18.192 7.92L32.498 27 18.192 46.08a1 1 0 0 0 .2 1.4l1.6 1.2a1 1 0 0 0 1.4-.2L36.597 28.2a2 2 0 0 0 0-2.4L21.392 5.52a1 1 0 0 0-1.4-.2l-1.6 1.2a1 1 0 0 0-.2 1.4z\\"></rnsvgpath></rnsvggroup></rnsvggroup></rnsvgsvgview></view></view></view></view></view></view><view style=\\"height: 0px;\\"></view></view></rctscrollview></rctnavitem></rctnavigator></view>"
`;
